### DISCUSSION TITLE
Lack of Controlled TimePicker and DatePicker Components

### DISCUSSION THREAD

**Discussion by qusaikg** [trussworks/react-uswds#2992] [General]
We currently use the `TimePicker` and `DatePicker` components in our app, but need the ability to control the `TimePicker` values from the parent component, such as `<TimePicker value="11:30" />`. The current implementation doesn't support this feature.

Additionally, we've observed that the onChange callback is triggered twice on initial render, which appears to be a bug.

Could we add support for controlled components and address the callback issue? Your assistance would be appreciated.

copy @werdnanoslen

**Comment by werdnanoslen** [comment]
Hi @qusaikg , would you please make two issues for this: a bug issue for the onChange callback and a feature request for the value setting?
